recipes-bsp/secure-boot-otp-provisioning/files/keys/custMpk.pem

```diff
@@ -0,0 +1,52 @@
+uzzPI0LmLGBry0A9V/vXvhDtvbDRlbHgKwUlQyUxvlBEUoQk3PFJ3LeYliZks95o
+GuDvu0Eo1pnrszyMITWequPts6qJib+kwwqjtKVihp7CpkXt4qkIORcEDw8eSSju
+BCs1ETEOwi/l9HZszFMIgU/RN3dHa2H1w8w9cbJ/XxA0ktIud+TO4mOLV59WINJA
+NNv+q6dh0LQxx5+BG5VoJGmzOVWn4zV1X07dQHkSfo49rKRnzgOeQ/R5tMlwu5fd
+oYp/3lo8OyU7zQ9IhkqBLoG9XJyOzcU1JCiF2+GJb+4oV5tEIDsEKjGwitl0hKs5
+0iVPy0zJxFOIDI5O5N0G1GJ+cNFJaRslgtMQkEegb7eyFY1x/yuq7ZwUcWJE8K9m
+EpgQD/JlIb2pFCejdeHOnseXutnx9QIDAQABAoICACxcyoEbWyM/eULfFsTaR2Uj
+iIouqHY4PdGsWB3Pkj1kdFxCn0WmD+PP6g/iw9qGsJ+CCD6C1AwxCiUaRMWHdnZq
+ghC/ffm8kY9n1nhDQkt9T9nMPBCfHEfjKRcbADeadmEOSlY6waIRhnD0dNI7wcM/
+YP4jREkOFrM8WixWcvqMMs4hXRu3msAunuUOYjBwkGa80o9/+oLQoVU/8waw/6gO
+f4UMbhwWvcJ1mg5AMiHyjvWdvdz0jYbCrAr5Nyp7nKTD9X2MULQxtfILk/+Y/cEi
+/Hw1CrBU/o34m8sw/nHyxkZ3gTLIF9sxFyfz26rlF8TKxZajE2niI9XSQnHPqjuG
+8yERhYuWlcR50vPLhi7BgXIL7TQgNFhZwrtKBx+wFswfoF9C+NAW1dsp+e37D+jC
+wu0T1no/TZRXoGZUTHGpY1Q9U2AovclcNPumujWvkTwKpKp5sKksNPRnhUMBqmxE
+I63BcdFVcyGvLYqRqusr9qltbcsYN/n8tPM3L1K7vxoUB5S9vA6yLi+YrG5CHoRI
+lEoJKry0A4kQdpX3NwOxY3+Qs3NjF/5bU4o/5OjJkH57D21SrbycJ7ExLXaOV8oh
+gb5uYEWPOn4HFhNCvoMyp9xf03846HFN29qnT+dKZoay9ovFe/a8ML+/6klOOO8R
+Lfc+VFRlArmTUm5GVOxBAoIBAQD3yAE0h+/GNvw67p9QCPND9H5FBOn4Br9wPXa9
+U/5777eKfW6XBvCQMRmreMH9E52d9+o44TqdFJPyQbyjSemlwNngFI6ZMNso5K1T
+QX5Txd1H5SrcGdmjYO1o1mtg5y3RHtuCVmoMc6iwjsSubDlNOygWFjZcC+QPwSIV
+eWLtcTEl7PeTcCqZl17yxhJKFtp2orp+Om8awqjIZozqrGkK/HXMF71rc6pQnZpD
+e/zQohJf3DDMrVK1Pk6ix026aHGaZ0Rm4p8zqUvdr3NF5RqVPhXHjSyMbFS8hD+F
+0mjhO+8+JHjOkKGmDX21xP4afTqa3t8/LnUKjWsJnPbo/ieRAoIBAQDHEssufy7W
+c0H+F5WAqRBzOZ+j4uBg4GNLID9+smtVlHp2TF+c6i6CSZ9hp024Nkh/HGHjmJ8p
+MKan6iFTTZXdTAw4Atni8QErHo0+AbVjKvqzCA3Rb1kVjsQa1aMwvBCHptBSbKeS
+K1ktHr/MOQFYAPaA3bSMul+JrSmAFcvEu8rV0+spU3+KryWLsrd5RiypkzTIlVSu
+qTuqvVfPQF1ETKRUrA3n3QZxr7UggTHbnPKcv3ZO1ylT+cgRTy8uj0TRRxWVGZOA
+0GyJeHRCFcR/sbZ6jMYY3cCOhI2i6OJt5nwoCln1yYHmWuxFPbtOCHocyANKb60z
+Tk4jCT72wZolAoIBAQC2MTVZp/7Rs8qhlqZEFmCxfgx+uY9EVcncQQWXTzSXcvt1
+dKelm2+1xrTGVcPmT88cwZdS/aG4sNHcVyPauVV5EKXj05nK4Ja1qfgceNLnzFza
+KZLrRoK2VGYdKVTJaxNQai0j2GCgCUdnk4LhcihsNmxSWUDdS80FHJDwU1I/n3Rx
+G54rplVcnGYvgx8dpVI724nfmR3rAmlS+bzyLZYLgvkm+enJIcKBBnHQ31FVI2b5
+X5HuXh+rr0XpAT2RLI0j6HzuXgKL3RcwyXqsUTrLtFwVYCP6mQlxmJiQ0NdXjldr
+TSwbT1PdIpxl1JS3a491Ix3qekGLYdLrvaP8TC1RAoIBACI7wyD90goiy+hsGdOE
+P0XxVNiT/U3riGe05fZdrEONQvnw7EhJpTHWg0gBw0W4YAMr2FGRP12P9jjN9Gkk
+4G6r0zYWl22FUFQf7t4I4S5qh64d5nJrmYAxituqfWO2E4mgUBa8hJyIQ8+UiBsQ
+eoCtubYzagcBTciZNJfIJ8BvItRemwQCTd6FfAln9FEwliGbJOqRc3yNsasM4HrN
+cLx8CY5eAA4tXhxSwO0UCoTfmoEGE8w/B+Ze/96qJUz2ajkq5jU7rrfqH6AMTMHI
+iIAFCYJNjwtbpu9bMBghwD5x8kvYa9vuiR4Z6KOHmBw1LPqiEp7MUZaPErTHtEIu
+zNkCggEAcLWGaQAgarwBvBZZhSuQPsT0MsY2weML/C4ebYxiqx6kGHFEQqSo+a30
+xWFN8XlAxi99BU/HdoGjoMOQ3VX/Czm0Ruo2QyEM3p7HIagXDhbi8LBxTwX0RuPR
+jHw4lzLT8Eg4NKKdVD0sPsQunZioHE+Tw//NXFEzVjhuMmEHM1iNB6wM2GYWTS6C
+KJcPKVsI3boGe3VVqdQJSXlymHmVMuw063B7pUeJc++PSOTzQWgF0rc6vDnRceGP
+wobxa7qectmRYhzuaFvPYO6Vaf3HyBij3psrOHEjun1Enph4/pPoGPh1LSosyDPM
+nLdZ5bI2pL9G4VAC4WxE8EEIFJONnA==
+-----END PRIVATE KEY-----
```

Comment:
We already have custMpk.* in-tree. Is this a copy or even another version?

Reply:
A copy, could be remodel to symbolic. However, user could use different keys.

Reply:
Not, not for signing the image. Another reason to avoid messing around with yet another set of keys and yet another location in the tree.